modules/contest-2021/ContestHero.tsx

```diff
@@ -116,14 +130,24 @@ const ContestHero = (): ReactElement => {
               className={classes.buttonCta}
               variant="raised"
               color="secondary"
-              onClick={openDialog}
+              component="a"
+              href="#contest-register"
+              // onClick={openDialog}
```

Comment:
Remove uncommented code?

Reply:
You cool with keeping this here? We're going to be switching back and forth between a link and a dialog for the CTA frequently, kept it for my utter laziness 😇